django/db/migrations/loader.py

```diff
@@ -6,11 +6,15 @@
 
 from django.apps import apps
 from django.conf import settings
+from django.db import connection as connection_
```

Comment:
What @shaib said: `makemigrations` and `migrate` should fail, `showmigrations` should work.